Source/Dafny/Compiler-java.cs

```diff
@@ -685,26 +689,31 @@ protected class ClassWriter : IClassWriter {
         if (ComplicatedTypeParameterForCompilation(argType)) {
           Error(tok, "compilation of set<TRAIT> is not supported; consider introducing a ghost", wr);
         }
+
+        if (erased) return DafnySetClass;
```

Comment:
Surround the then branch with curly braces. (Same comment below.)

Reply:
Fixed